design/prot.txt

```diff
@@ -141,6 +153,39 @@ _`.impl.w3`: Windows implementation.
 
 _`.impl.xc`: macOS implementation.
 
+_`.impl.xc.prot.exec`: The approach in `.sol.prot.exec`_ of always making
+memory executable causes a difficulty on macOS on Apple Silicon. On
+this platform, programs may enable `Hardened Runtime
+<https://developer.apple.com/documentation/security/hardened_runtime>`_.
+This feature rejects attempts to map or protect memory so that it is
+simultaneously writable and executable. Moreoverm the feature is
+enabled by default in recent versions of Xcode, so that if you install
```

Comment:
A version (e.g. "as of macOS 13 Ventura") would be more future-proof than "recent".